### PR TITLE
Creating package.json file

### PR DESCRIPTION
**Pull request by fmoliveira**
Creating package.json flie to declare Node.js dependencies.

This will make it easier for anyone wanting to contribute with the project to get started.

### CODE REVIEW

**Comment by fmoliveira** [comment]
Resolves #44 .

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "bootstrap-wysiwyg",
+  "version": "1.0.3-beta",
```
Comment (steveathon) [inline package.json]: Looks like this might throw an error now that we've released a new version. I' can probably update that and re-release.

Comment (fmoliveira) [inline package.json]: You mean that it'll be harder to maintain the version number in two different files?

Perhaps you could create a new gulp task with [gulp-bump](https://www.npmjs.com/package/gulp-bump) to bump versions from bower.json and package.json simultaneously.

**Comment by RandomlyKnighted** [comment]
@fmoliveira I'm not sure I understand the purpose of this PR. There's already a [package.json file](https://github.com/steveathon/bootstrap-wysiwyg/blob/master/package.json) in the master branch.

**Comment by fmoliveira** [comment]
You're right, sorry. I must have messed up with Mindmup's repo, which doesn't have the file.

I'm so sorry for the lack of attention.

I'm closing this. Thanks very much.

**Pull request closed without merging**